ibis/backends/base/sql/alchemy/client.py

```diff
@@ -329,3 +330,101 @@ def _sqla_table_to_expr(self, table):
     def version(self):
         vstring = '.'.join(map(str, self.con.dialect.server_version_info))
         return parse_version(vstring)
+
+    def insert(
+        self,
+        table_name: str,
+        obj=None,
```

Comment:
how can this ever be None? pls fully type this. Why allow a string at all? that is not supported anywhere else in the ibis api.

Reply:
let's start really simple and just take a DataFrame or an Expr

Reply:
Done!